_schemaV2.graphql

```diff
@@ -3244,6 +3247,7 @@ type CalendarYearMarketPriceInsights {
   createdAt: ISO8601DateTime
   id: ID!
   lotsSold: BigInt
+  medianSalePrice: BigInt
```

Comment:
@lordkiz this will add the `medianSalePrice` to `CalendarYearMarketPriceInsights` and you can switch from `averageSalePrice` to `medianSalePrice` as soon as this is merged.